onfleet/onfleet.py

```diff
@@ -178,6 +178,20 @@ def __call__(self, *args, **kwargs):
                 error_code = error_data['error']
                 error_message = error_data['message']
 
+                options = parse_options(error_cause)
+
+                if options:
+                    # If the options regex returned some options, this must be
+                    # a multiple destination issue. This is the only way to
+                    # tell since onfleet doesn't use distinct error codes
```

Comment:
I'd like a bit more certainty in this. I don't mind "gross" in the sense that we're parsing a string because the information we want isn't passed back in a structured way in the API. But I do mind "brittle" in the sense that if they change their response format slightly we might accidentally start raising MultipleDestinationErrors for everything. 

Can we make sure that if an API structure change breaks this, that it breaks it in a way that doesn't disrupt the normal functioning of all the non-multiple-destination cases? Maybe it already is, but without an example response text and some clarity on what the regex is searching for, it's tough for me to tell. Might be worth including that stuff here in the code, actually.